drush/Commands/NewRelicCommands.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Drush\Commands;
+
+use Consolidation\AnnotatedCommand\CommandData;
+use Drush\Commands\DrushCommands;
+
+class NewRelicCommands extends DrushCommands
+{
+
+  /**
+   * @hook option *
+   *
+   * @option nr-name New Relic transaction name.
```

Comment:
@weitzman Will the option need to be `--nr-name`, rather than `--nrname` described in the description, with this set?

Reply:
Fixed